saml/service-providers/bioland-dev/profile-mapper.js

```diff
@@ -0,0 +1,57 @@
+import { name, uid, email, givenname as firstName, surname as lastName, } from "../../services/saml-constants/claims.js"
+import { unspecified } from "../../services/saml-constants/name-identifer-formats.js"
+import _ from "lodash"
+
+const ZERO_WIDTH_JOINER = '\u200D';
+
+export default class  {
```

Comment:
maybe we can delte this one if defualt makker correctly map userID to uid? (see my comemt on default-profile-mapper.js)